lib/src/specs/icon/icon_spec.dart

```diff
@@ -7,12 +7,39 @@ import 'icon_attribute.dart';
 class IconSpec extends Spec<IconSpec> {
   final Color? color;
   final double? size;
+  final double? weight;
+  final double? grade;
+  final double? opticalSize;
+  final String? semanticLabel;
```

Comment:
Its important to only have visual semantics within the spec, I believe semanticLabel is passed down directly to the widget.